api/pkg/apis/v1alpha1/utils/symphony-api.go

```diff
@@ -204,12 +205,22 @@ func SyncActivationStatus(context context.Context, baseUrl string, user string,
 
 	return nil
 }
-func ReportCatalogs(context context.Context, baseUrl string, user string, password string, instance string, components []model.ComponentSpec) error {
+func ReportCatalogs(context context.Context, baseUrl string, user string, password string, catalog string, components []model.ComponentSpec) error {
 	token, err := auth(context, baseUrl, user, password)
 	if err != nil {
 		return err
 	}
-	path := "catalogs/status/" + url.QueryEscape(instance)
+	var name string
+	var version string
+	parts := strings.Split(catalog, ":")
```

Comment:
We are mixing responsibility of separate microservices. Catalogs vendor should not interpret version strings as they don't have versioning logic. CatalogContainer can handle these versioning strings. Now we are mixing the responsibility of two services into one. We can't do this.